hashes/contrib/test.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
This should be `/usr/bin/env bash`. Not all systems have `bash` in `/bin`.

Reply:
Sure thing, can do.